fakes/image.go

```diff
@@ -67,6 +67,10 @@ func (i *Image) Label(key string) (string, error) {
 	return i.labels[key], nil
 }
 
+func (i *Image) Labels() (map[string]string, error) {
+	return i.labels, nil
```

Comment:
The fake image never has `nil` labels, and therefore code that passes tests against the fake image might be vulernable to nil pointer exceptions when provided a local or remote image at runtime. We should either change the local and remote images to return an empty map or change the fake image to return `nil`.

Reply:
See previous response, I don't think we are in danger of a pointer exception here. `nil` behaves like an empty map already.